internal/statemachine/classic_states.go

```diff
@@ -562,6 +563,14 @@ func (stateMachine *StateMachine) installPackages() error {
 	var classicStateMachine *ClassicStateMachine
 	classicStateMachine = stateMachine.parent.(*ClassicStateMachine)
 
+	// copy /etc/resolv.conf from the host system into the chroot
+	if !osutil.FileExists(filepath.Join(classicStateMachine.tempDirs.chroot, "etc", "resolv.conf.tmp")) {
```

Comment:
Suggestion: could we move the check for this into the `helper.BackupAndCopyResolvConf`? It will remove some boilerplate and I think it really should be done automatically everytime we want to backup and copy resolv.conf.